docs/books/learning_ansible/04-ansible-galaxy.md

```diff
@@ -11,7 +11,7 @@ In this chapter you will learn how to use, install, and manage Ansible roles and
 **Objectives**: In this chapter you will learn how to:
 
 :heavy_check_mark: install and manage collections;       
-:heavy_check_mark: install and manage roles;.   
+:heavy_check_mark: install and manage roles.   
 
```

Comment:
We are back to that ";" again. The ";" is fine I think, just not with the ";." as shown. If you correct it to be a period, then all of the items should also be ended with a period. IMO

Reply:
@sspencerwire please review